Dockerfile

```diff
@@ -1,10 +1,14 @@
 FROM python:3
 
-WORKDIR /usr/src/app
+WORKDIR /workdir
 
 COPY requirements.txt ./
 RUN pip install --no-cache-dir -r requirements.txt
 
-COPY . .
+RUN groupadd -r pelican && useradd --no-log-init -r -g pelican pelican
```

Comment:
Should we just use a generic name like `builder` instead of app-specific names like `pelican`, so that it's easier to compare Dockerfile's across projects?

Reply:
Changed to `runner` and applied the other conventions